### PR TITLE
PLNSRVCE-658: switch from configmamp to systemconfig

### PR DESCRIPTION
**Pull request by gabemontero**

### CODE REVIEW

**Comment by gabemontero** [comment]
after this is merged @stuartwdouglas I can switch back to kustomize patch merge with a ref to image-config.yaml over in https://github.com/redhat-appstudio/jvm-build-service/blob/819aababfec519b14959e40eb0fe3f0893b455b4/deploy/operator/config/system-config.yaml#L21

File: image-config.yaml [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 metadata:
-  name: jvm-build-system-config
-  namespace: jvm-build-service
-data:
+  name: cluster
+spec:
   #This is used to drive the builder image build
   #Versions places here for gradle will be used generate the relevant dockerfiles
   #after you run generate.sh
-  builder-image.jdk8.tags: jdk:8,maven:3.8,gradle:7.4.2;6.9.2;5.6.4;4.10.3
```
Comment (stuartwdouglas) [inline image-config.yaml:11]: generate.sh needs to be updated as per the comment above.

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]
I will have a look at fixing this today

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**